examples/with-mocha/src/Child.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react'
+
+function Child() {
+  const [fetched, setFetched] = useState(false)
+  const [result, setResult] = useState()
+  useEffect(() => {
+    fetch('http://example.com/api/getChildList', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        pagination: {
+          pageSize: 10,
+          pageNo: 0,
+          sortedColumn: 'createdBy',
+          sortedType: 'asc',
+        },
+      }),
+    })
+      .then(async (response) => {
+        const result = await response.json()
+        console.log('Fetch complete: ', result)
```

Comment:
Looks like a left-over after debugging. Should we remove this `console.log`?

Reply:
Yea, agreed.